ops/print-etcd-cheatsheet.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+function get_wrapper_process_id() {
+  process_id=$(pgrep wrapper)
+  echo "${process_id}"
+}
+
+function get_backup_restore_process_id() {
+  process_id=$(pgrep etcdbrctl)
+  echo "${process_id}"
+}
+
+wrapper_pid=$(get_wrapper_process_id)
+backup_restore_pid=$(get_backup_restore_process_id)
```

Comment:
Verify that both `wrapper_pid` and `backup_restore_pid` are valid before proceeding